README.md

```diff
@@ -1 +1,130 @@
-# my-app-calculator tests
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <br/>
+
+  <h3><b>Calculator React project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone Project: calculator <a name="about-project"></a>
+
+This project is meant to create a SPA with react and implement the functions of a calculator
+
+[🎥 Check out this video of me explaining the key features of the project](https://mega.nz/file/T6B0UDKK#sGkcTmkP2iN5-36AmX7X1eUJa38Olgx0h2M7537y1mg)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+-React
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **React**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Coming Soon <a name="live-demo"></a>
+
+- [Live Demo Link Coming Soon](#)
```

Comment:
- [ ] It would be better to remove the misleading live demo link, this makes our README file no professional